pkg/driver/node.go

```diff
@@ -100,10 +100,24 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 		return nil, status.Errorf(codes.Internal, "Could not create dir %q: %v", target, err)
 	}
 
-	klog.V(5).Infof("NodePublishVolume: mounting %s at %s with options %v", source, target, mountOptions)
-	if err := d.mounter.Mount(source, target, "lustre", mountOptions); err != nil {
-		os.Remove(target)
-		return nil, status.Errorf(codes.Internal, "Could not mount %q at %q: %v", source, target, err)
+	//klog.V(5).Infof("NodePublishVolume: mounting %s at %s with options %v", source, target, mountOptions)
```

Comment:
Can just delete this instead of commenting out and leaving it here

Reply:
done, thanks for pointing out